ci/provision-acs-interface.sh

```diff
@@ -8,10 +8,12 @@ sudo pip3 install pipenv
 cd /vagrant
 pipenv install --ignore-pipfile 2> /dev/null
 mkdir -p data
+cp ./examples/.env .
```

Comment:
You  sure this is ok? It will overwrite your local `.env` file when using a shared folder. The readme now encourages the user to make their own `.env` file.

Reply:
My bad!